static/sass/components/_button.scss

```diff
@@ -66,14 +66,14 @@
 }
 
 .mzp-t-leave.mzp-t-product {
-  background-color: $color-red-60;
-  color: white;
-  border-color: $color-red-60;
+  background-color: transparent;
+  border: 2px solid var(--color-red-60);
```

Comment:
Why not use the sass variables for the colors?

Reply:
Transparent is kind of odd one out because its value would not change. Other places also refers it as transparent in this file.